code/game/atoms_movable.dm

```diff
@@ -289,6 +289,7 @@
 	locking_categories += C
 
 /atom/movable/proc/get_lock_cat(var/category = /datum/locking_category)
+	locking_init()
```

Comment:
You shouldn't intialised a list here either. If it's just initialised, it's always going to return null